posts/2019-07-29-recordwildcards.md

```diff
@@ -167,10 +166,10 @@ fields like in the code below:
 defaultUser :: User
 defaultUser =
     let userName = "Ivan"
-    in User{..}
+     in User{..}
```

Comment:
I think this one is not a typo. There should be 4 spaces before each line if I counted properly :slightly_smiling_face: 
```suggestion
    in User{..}
```

Reply:
Yes, sorry, this is my ormolu OCD acting on me 😂